README.md

```diff
@@ -1,44 +1,38 @@
+
+## Contents
+
+In this guide, we will describe 5 ways to make your C++ code available as a web application or web wervice
 
-- as a cgi script
-- as a Python application via pybind11, Flask and Celery
-- in the web browser using web assembly and JavaScript
+1. [Web service using Common Gateway Interface](#method-15-web-service-using-common-gateway-interface)
+1. [Python web service using Swagger/OpenAPI](#method-25-python-web-service-using-swaggeropenapi)
+1. [Python web application using Flask](#method-35-python-web-application-using-flask)
+1. [JavaScript web service using Emscriptem and WebAssembly](#method-45-javaScript-web-service-using-emscriptem-and-webassembly)
+1. [JavaScript web application with React](#method-55-javaScript-web-application-with-react)
 
-This guide was written and tested on Linux operating system. The required dependencies to run this guide are described in the [INSTALL.md](INSTALL.md) document. If you want to contribute to the guide see [CONTRIBUTING.md](CONTRIBUTING.md). The [repo](https://github.com/NLESC-JCER/cpp2wasm) contains the files that can be made from the code snippets in this guide. The code snippets can be [entangled](https://entangled.github.io/) to files using any of [these](CONTRIBUTING.md#tips) methods.
+This guide was written and tested on Linux operating system. The required dependencies to run this guide are described
+in the [INSTALL.md](INSTALL.md) document. If you want to contribute to the guide see [CONTRIBUTING.md](CONTRIBUTING.md).
+The [repo](https://github.com/NLESC-JCER/cpp2wasm) contains the files that can be made from the code snippets in this
+guide. The code snippets can be [entangled](https://entangled.github.io/) to files using any of
+[these](CONTRIBUTING.md#tips) methods.
 
-The [Newton-Raphson root finding algorithm](https://en.wikipedia.org/wiki/Newton%27s_method) will be the use case.
-The algorithm is explained in [this video series](https://www.youtube.com/watch?v=cOmAk82cr9M).
-The code we are using came from [geeksforgeeks.org](https://www.geeksforgeeks.org/program-for-newton-raphson-method/).
+## Example program: Newton-Raphson root finding
 
-Let's first define the mathematical equation, which we will be searching for its root, and the derivative of it.
+The [Newton-Raphson root finding algorithm](https://en.wikipedia.org/wiki/Newton%27s_method) will be the use case. The
+algorithm is explained in [this video series](https://www.youtube.com/watch?v=cOmAk82cr9M). The code we are using came
+from [geeksforgeeks.org](https://www.geeksforgeeks.org/program-for-newton-raphson-method/).
 
-The equation and its derivative which we will use in this guide are $x^3 - x^2  + 2$ and $3x^2 - 2x$ respectively.
-The root is the value (X-coordinate) which makes the mathematical function (Y-coordinate) equal to `0`. In this equation the root is `-1`.
+Let's first define the mathematical function and its derivative, which we need in order to find the root.
```

Comment:
Why did you rename equation back to function?

Reply:
I didn't realize that was what it said before. I renamed it because it more precisely captures the meaning of the dollared content. Also, 'derivative' makes more sense for a function than for an equation.